prime_server/http_protocol.hpp

```diff
@@ -191,3 +195,6 @@ struct http_response_t : public http_entity_t {
 using http_server_t = server_t<http_request_t, http_request_info_t>;
 
 } // namespace prime_server
+
+prime_server::shortcircuiter_t<prime_server::http_request_t> make_shortcircuiter(const uint8_t& verb_mask,
```

Comment:
this should be within the namespace and also have defaults of all verbs and maybe an empty healthcheck path

Reply:
Done